bears/css/PostCSSBear.py

```diff
@@ -0,0 +1,27 @@
+from coalib.bearlib.abstractions.Lint import Lint
+from coalib.bears.LocalBear import LocalBear
+from coalib.settings.Setting import typed_list
+
+
+class PostCSSBear(Lint, LocalBear):
+    executable = "postcss"
+    gives_corrected = True
+
+    def run(self, filename, file, postcss_plugins: typed_list(str)=[]):
```

Comment:
why do you give an empty list as default and then error out? Just give no default so the user will be asked